ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_permissions_mgmt.robot

```diff
@@ -189,17 +189,17 @@ Restore Permissions Of The Project
 Refresh Pages
     Switch To User    ${USER_A}
     Open Data Science Projects Home Page
-    Reload RHODS Dashboard Page    expected_page=Data science projects
+    Reload RHODS Dashboard Page    expected_page=Data Science Projects
```

Comment:
It would probably make sense to create a constant for this value and use it an all these places.